README.md

```diff
@@ -17,6 +17,20 @@
 
 ## Packages
 
+### Versioning
+
+When a component reaches v1.0.0., it is considered to be stable and will follow [SemVer](http://semver.org/) for versioning.
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs and related issues/PRs for more information
```

Comment:
Nit: Consult the component CHANGELOGs, related issues, and PRs for more information.